ckanext/dgu/theme/templates/page.html

```diff
@@ -15,18 +15,16 @@
 #}
 {%- block content %}
     {% block maintag %}<div role="main">{% endblock %}
-      <div id="content">
-        {% block main_content %}
```

Comment:
Ah, that helps - I've got it now.

Just so it is noted: Core ckan has the toolbar (breadcrumbs & flash messages) styled the same as the content below it i.e. with the same background etc., whereas DGU has the toolbar content on a grey bar of its own. That is achieved if the toolbar is not under #content, which is what you've done.
Keeping toolkit in `#content`:
![screen shot 2015-01-20 at 15 12 20](https://cloud.githubusercontent.com/assets/307612/5819595/f119018e-a0b6-11e4-8081-d4585ca28b8d.png)
Moving toolkit out (as it should be):
![screen shot 2015-01-20 at 15 12 51](https://cloud.githubusercontent.com/assets/307612/5819598/f3cd0510-a0b6-11e4-950c-5c0d420e9482.png)